src/renate/updaters/learner.py

```diff
@@ -404,14 +402,14 @@ class ReplayLearner(Learner, abc.ABC):
     def __init__(
         self,
         memory_size: int,
-        memory_batch_size: int = defaults.BATCH_SIZE,
+        batch_memory_frac: float = defaults.BATCH_MEMORY_FRAC,
         buffer_transform: Optional[Callable] = None,
         buffer_target_transform: Optional[Callable] = None,
         seed: int = defaults.SEED,
         **kwargs,
     ) -> None:
         super().__init__(seed=seed, **kwargs)
-        self._memory_batch_size = min(memory_size, memory_batch_size)
+        self._batch_memory_frac = min(batch_memory_frac, memory_size / self._batch_size)
```

Comment:
and here we could check that `memory_size > batch_memory_frac * self._batch_size`

Reply:
Yes, I'll add that.